### ISSUE TITLE
Mobs attracted to Train Tracks

### ISSUE THREAD

**Issue by iancbogue** [Mrbrutal/Traincraft-5#66]
Villagers and animals seem to be physically attracted to the tracks. Villagers will hop over the tracks, while the animals will just linger around waiting to be killed by a passing freight train or high speed passenger train.

**Comment by EternalBlueFlame** [comment]
This is an issue primarily related to tile entities. In many cases vanilla rails will do the same thing.
The issue seems to be more common in Cauldron/CKauldron/Thermos servers.

**Comment by iancbogue** [comment]
OK. So no solution then, other than blocking off the tracks?

**Comment by EternalBlueFlame** [comment]
Yeah blocking off the tracks is about your only option in most cases. if the blocks in the tile entities are made to be a half block high that can prevent them from getting stuck, but the attraction is just something random and dumb that happens from the AI code.
